test/addons/layout-change/lib/test-cuddlefish-loader.js

```diff
@@ -21,11 +21,6 @@ exports["test compatibility"] = function(assert) {
   assert.equal(require("tabs"),
                require("sdk/tabs"), "sdk/tabs -> tabs");
 
-  if (app.is("Firefox")) {
-    assert.equal(require("widget"),
-                 require("sdk/widget"), "sdk/widget -> widget");
-  }
```

Comment:
> what we should test?

    // sdk/widget module was removed
    assert.throws(() => require("widget"), /.../, "There is no widget module");
    assert.throws(() => require("sdk/widget"), /.../, "There is no sdk/widget module");

> So what should be the purpose of the test? You're suggesting to test that every module we remove it's actually failing in requiring them?

Every high level module yes.

First of all we've never removed a high level module in the past, so this is a unique case.  I'm suggesting we should test that every high level module we remove is actually gone, so that we don't add it again either on purpose or by mistake, and cause regressions.

> And moreover, where we should add those tests?

I don't think this matters, we can make a new test file for it or use an old one.